### PR TITLE
Adding automated tests

### PR DESCRIPTION
**Pull request by bbaassssiiee**

### CODE REVIEW

**Comment by igor-nikiforov** [comment]
Hi @bbaassssiiee! Huge thank you for your contribution. I really appreciate your work.

Just 2 minor comments and we LGTM here.

**Comment by bbaassssiiee** [comment]
Please don't forget to submit your review, only then I can see your comments...

File: defaults/main.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 etcd_version: "3.4.13"
+etcd_archive: "etcd-v{{ etcd_version }}-linux-amd64.tar.gz"
+etcd_download_url: "https://github.com/etcd-io/etcd/releases/download"
```
Comment (bbaassssiiee) [inline defaults/main.yaml:4]: This allows on-prem use.

File: defaults/main.yaml [hunk 2]
```diff
@@ -9,8 +11,9 @@ etcd_conf_dir: "/etc/etcd"
 
 etcd_config: {}
 
-etcd_service_enabled: True
```
Comment (bbaassssiiee) [inline defaults/main.yaml:12]: yamllint default Boolean value

File: handlers/main.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
-- name: restart etcd
+- name: Restart etcd
```
Comment (bbaassssiiee) [inline handlers/main.yaml:2]: ansible-later CheckNameFormat

File: .later.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+ansible:
+  # Add the name of used custom Ansible modules. Otherwise ansible-later
+  # can't detect unknown modules and will through an error.
+  # Modules which are bundled with the role and placed in a './library'
+  # directory will be auto-detected and don't need to be added to this list.
+  custom_modules: []
+
+  # List of allowed literal bools (ANSIBLE0014)
+  literal-bools:
```
Comment (bbaassssiiee) [inline .later.yml:10]: compatible with yamllint and JSON

File: .yamllint [hunk 5]
```diff
@@ -0,0 +1,10 @@
+---
+extends: default
+
+rules:
+  truthy:
+    allowed-values: ['true', 'false']
+    check-keys: false
```
Comment (bbaassssiiee) [inline .yamllint:7]: This is for the Github action

File: tasks/verify.yml [hunk 6]
```diff
@@ -0,0 +1,29 @@
+---
+# This is to execute tests.
```
Comment (bbaassssiiee) [inline tasks/verify.yml:2]: called by molecule verify

**Comment by bbaassssiiee** [comment]
Please tell me what's keeping up the merge...

**Comment by igor-nikiforov** [comment]
@bbaassssiiee sorry for silence. I was on vacation during previous month an was not able to completely deep dive into this PR.

**Pull request merged**